apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/PlatformAPIWebview.tsx

```diff
@@ -230,7 +230,7 @@ export const PlatformAPIWebview = forwardRef<WebviewAPI, WebviewProps>(
                   signature,
                   exchangeType,
                   swapId,
-                  rate,
+                  amountExpectedTo,
```

Comment:
Are you sure you want to modify the exchange part of the old platform api webview instead of the wallet-api webview ?

Reply:
You are right, removed it